src/commands/use.js

```diff
@@ -58,10 +58,10 @@ module.exports = new Command("use [alias_or_project_id]")
     if (newActive) {
       // firebase use [alias_or_project]
       var aliasedProject = options.rc.get(["projects", newActive]);
-      return api.getProjects().then(function(projects) {
+      return firebaseApi.getProject(newActive).then(function(project) {
         if (aliasOpt) {
           // firebase use [project] --alias [alias]
-          if (!projects[newActive]) {
+          if (!project) {
```

Comment:
Did you test this against a non-existent project? I think it might actually reject the promise as opposed to resolving with a null here.

Reply:
Tried it out, the command errors with "Error: HTTP Error: 403, Permission denied or not found." instead of following the logic here.

Try this:

```js
var project = null;
return firebaseApi.getProject(newActive)
  .then(foundProject => {
    project = foundProject;
  })
  .catch(() => {})
  .then(() => {
    // ... continue code from before
  });
```

Reply:
Thanks for these tips they were very helpful. 

I have refactored the code a bit as I realized I was not properly handling alias commands. Things should work as expected now including handling the 403 error.